### PR TITLE
Add helper commands reference; Add roles stylesheet

### PR DESCRIPTION
**Pull request by rattus-rattus-rattus**
1. New page `/helper-commands` added to document all helper commands and factoids.
2. New stylesheet under `extra_stylesheets` containing colors for roles mentions matching those server-side. Also includes hover styles.

### CODE REVIEW

File: helper-commands.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Marks a forum post as solved. Can be used by the previous owner or by people with the <a class="mention helpful">@Helpful</a> role.
+
+### `.unsolved`
+
+Mark a forum post as unsolved. Can be done by the owner of the thread or by people with the <a class="mention helpful">@Helpful</a> role.
+
+## Factoids
+
+A factoid is a short command that can be used to prompt <a class="mention">@Bot</a> to recall a single message containing text. To use a factoid, simply send a message containing the factoid command and no other text. Below is a list of all help-related factoids, alongside their content. These can be used by all users.
```
Comment (mniip) [inline helper-commands.md]: Factoid names can be followed by other things. Here's an excerpt from the lore document:

> The bot contains a configurable library of “factoids” — canned responses to common inquiries. Every factoid consists of a name, by which it is triggered, and bot’s response, which can be either text or an embed. Factoids can also have aliases, where multiple names refer to the same response.
Factoids can be triggered by a command-like syntax: `!<name of factoid>` . The name can be followed by anything, the bot will choose the longest prefix that matches a known factoid.

File: helper-commands.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Factoids
+
+A factoid is a short command that can be used to prompt <a class="mention">@Bot</a> to recall a single message containing text. To use a factoid, simply send a message containing the factoid command and no other text. Below is a list of all help-related factoids, alongside their content. These can be used by all users. 
+
+### `!15m`
+> Please only use the Helpers ping once if your question has not been answered for 15 minutes. Please do not ping or DM individual users about your question.
+
+### `!help`
+> To ask for mathematics help on this server, please open your own help channel or help thread. See ⁠#how-to-get-help for instructions.
```
Comment (mniip) [inline helper-commands.md]: Channel link?

File: helper-commands.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+### `.unsolved`
+
+Mark a forum post as unsolved. Can be done by the owner of the thread or by people with the <a class="mention helpful">@Helpful</a> role.
+
+## Factoids
+
+A factoid is a short command that can be used to prompt <a class="mention">@Bot</a> to recall a single message containing text. To use a factoid, simply send a message containing the factoid command and no other text. Below is a list of all help-related factoids, alongside their content. These can be used by all users. 
+
+### `!15m`
+> Please only use the Helpers ping once if your question has not been answered for 15 minutes. Please do not ping or DM individual users about your question.
```
Comment (mniip) [inline helper-commands.md:32]: So then do we update this page every time we update factoids?

File: helper-commands.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+> **If the original problem is not in English, then post it anyway!** The additional context might still help helpers help you. Do your best to translate.
+
+### `!da2a`
+> No need to ask "Can I ask...?" or "Does anyone know about...?"—it's faster for everyone if you just ask your question! See https://dontasktoask.com/
+
+### `!noping`
+> Please do not ping individual helpers unprompted.
+
+### `!nogpt`
+> Please do not trust ChatGPT or similar AI tools for mathematical tasks, as they often generate output which "sounds correct" but has numerous factual or logical errors. Use of these AI tools to answer other people's help questions is strictly against server rules (see #rules).
```
Comment (mniip) [inline helper-commands.md]: Channel link? Also potentially link to `/rules`

File: helper-commands.md (hunk 3 above)
Comment (rattus-rattus-rattus) [reply helper-commands.md:32]: That is the intention - helper factoids are added fairly infrequently so I think this is reasonable.

File: helper-commands.md (hunk 1 above)
Comment (rattus-rattus-rattus) [reply helper-commands.md]: > To use a factoid, simply send a message containing the factoid command and no other text.

to

> Factoids can be triggered by a command-like syntax: `!<name of factoid>`. The name can be followed by anything, and the bot will choose the longest prefix that matches a known factoid.

File: helper-commands.md (hunk 4 above)
Comment (rattus-rattus-rattus) [reply helper-commands.md]: Added `#rules` mention - this links to `/rules` rather than the channel

**Pull request merged**